### DISCUSSION TITLE
xref value is same for the different images

### DISCUSSION THREAD

**Discussion by YashMistry349** [pymupdf/PyMuPDF#1483] [Q&A]
Hey, 
I am using `page.get_image_info(xrefs=True)` to get the information of images present on the page, but when I check the Xref value which I get using `page.get_image_info(xrefs=True)` is the same for all image but bounding boxes are different.

Output of `page.get_image_info(xrefs=True)`
![image](https://user-images.githubusercontent.com/79560917/147063288-4c44fe75-31f9-404d-b378-c0eca6aacc1d.png)

Output of `page.get_images()`
![image](https://user-images.githubusercontent.com/79560917/147063405-c3e4ec0d-ada2-42a9-9f95-e861315fdc99.png)

To regenerate the issue I shared the sample pdf on your mail: jorj.x.mckie@outlook.de

Please do needful

Thank you

**Comment by YashMistry349** [comment]
@JorjMcKie Can you please check the above-mentioned issue? So that I can work on it as soon as it gets resolved.
Thanks

**Comment by JorjMcKie** [comment]
The two images are identical - whether or not the same image has been inserted twice.
The logic that associates an image display with an xref works by binary image content (via `Pixmap.digest`), so it will pick the xref of the first matching item in `page.get_images()`.
It otherwise is completely legal to display the same image multiple times on the same or different pages - using the same or different reference names ("Img10", etc.).

**Reply by JorjMcKie** [reply]
There is no issue!

**Comment by YashMistry349** [comment]
Thank you for your response

Your response will really help me to proceed further.

Thank you
Yash Mistry
